src/packages.ts

```diff
@@ -40,37 +45,31 @@ export class Type
         this.label = type.label;
     }
 
+    isInternal() : boolean
+    {
+        return this.label != 'internal';
```

Comment:
This is `return this.label == 'internal';`

Reply:
Obviously ... thanks.

Reply:
👍